hadoop-hdfs-project/hadoop-hdfs-rbf/src/main/java/org/apache/hadoop/hdfs/server/federation/router/RouterClientProtocol.java

```diff
@@ -588,6 +589,19 @@ public long getPreferredBlockSize(String src) throws IOException {
     return rpcClient.invokeSequential(locations, method, Long.class, null);
   }
 
+  /**
+   * Use the combination of src locations and dst to return RemoteParam.
+   */
+  private RemoteParam getRenameDestWithTrash(
```

Comment:
This method doesn't seem to have anything special regarding the trash.
I can also be static.

Reply:
It can also be static.